optimum/exporters/openvino/model_configs.py

```diff
@@ -2963,6 +2963,21 @@ class GLMOpenVINOConfig(LlamaOpenVINOConfig):
     MIN_TRANSFORMERS_VERSION = "4.46.0"
 
 
+@register_in_tasks_manager(
+    "glm4",
+    *[
+        "feature-extraction",
+        "feature-extraction-with-past",
+        "text-generation",
+        "text-generation-with-past",
+        "text-classification",
+    ],
+    library_name="transformers",
+)
+class GLM4OpenVINOConfig(LlamaOpenVINOConfig):
+    MIN_TRANSFORMERS_VERSION = "4.51.3"
```

Comment:
is there a specific reason why the patch release 4.51.3 is needed here ?

Reply:
GLM4 modeling is added since 4.51.3 https://github.com/huggingface/transformers/blob/v4.51.3/src/transformers/models/glm4/modeling_glm4.py